### DISCUSSION TITLE
How to listen to the change stream in the Milvus database.

### DISCUSSION THREAD

**Discussion by pmupkin** [milvus-io/milvus#27463] [Q&A and General discussion]
Currently, I would like to use Milvus to access change stream content, but upon a preliminary search, it seems that Milvus does not support this feature. I wanted to inquire whether Milvus currently supports change streams, or if there is a reference implementation available. I am currently using Milvus in Java.

**Comment by xiaofan-luan** [comment]
check this repo.

https://github.com/zilliztech/milvus-cdc

however this repo is still under heavy deployment and we will work on the document later

**Reply by pmupkin** [reply]
Thank you for your reply. I took a look at the linked content, and it appears that the CDC tool is written in Go. Does that mean if I want to use Milvus in Java, I'll have to wait for a Java version of the CDC tool to be updated in the future?
